pkg/drivers/powerflex.go

```diff
@@ -322,9 +322,11 @@ func ModifyPowerflexCR(yamlString string, cr csmv1.ContainerStorageModule, fileT
 			if env.Name == "X_CSI_MAX_VOLUMES_PER_NODE" {
 				maxVolumesPerNode = env.Value
 			}
-			if env.Name == "X_CSI_QUOTA_ENABLED" {
-				enableQuota = env.Value
-			}
+			/*
```

Comment:
Can we just remove this if we aren't going to use it?

Reply:
I'm a little torn on it, because I don't like leaving in commented code but the parameter was included at some point and it may be assumed to be included later. I suppose it'll be a defect at some point in the future, if someone expects the field to be there and it isn't. I'll remove it.